packages/block-editor/src/components/spacing-sizes-control/linked-button.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, Tooltip } from '@wordpress/components';
+import { link, linkOff } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+
+export default function LinkedButton( { isLinked, ...props } ) {
+	const label = isLinked
+		? sprintf(
+				// translators: 1. Type of spacing being modified (Padding, margin, etc).
+				__( 'Unlink %1$s' ),
```

Comment:
I think there might be room for improvement in the wording of this label:

- Adding "sides" to clarify what we are linking: "Link sides"/"Unlink sides"

Also, alternative wordings:
- "All sides"/"Individual sides"
- "Uniform sides"/"Custom sides"

I like "Uniform sides"/"Custom sides" because it helps me form a mental image of what might happen when I activate the linked mode.

cc. @richtabor

Reply:
I think "Unlink padding" and "Link padding" is ok (lowercase). 

It's not always the case that padding and margin are applied on all sides—generally horizontal and vertical—so uniform doesn't scale well. We could potentially adjust in the future though.

Reply:
fixed in eb85d052955c929051ae36d00c39c13459e0a13b